src/Desktop.Analyzers/UnitTests/AvoidDuplicateAcceleratorsTests.cs

```diff
@@ -1,10 +1,7 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Test.Utilities;
 using Microsoft.CodeAnalysis.UnitTests;
-using Roslyn.Test.Utilities;
-using Xunit;
```

Comment:
Oh, this is a pity -- removing `Xunit` from stub unit test files. Don't know what to suggest. I guess it's a small matter for the implementer to put it back, and this only comes up if your repo is full of stubs.